core/chains/evm/gas/models.go

```diff
@@ -154,6 +158,9 @@ func (e WrappedEvmEstimator) GetFee(ctx context.Context, calldata []byte, feeLim
 	if e.EIP1559Enabled {
 		var dynamicFee DynamicFee
 		dynamicFee, chainSpecificFeeLimit, err = e.EvmEstimator.GetDynamicFee(ctx, feeLimit, maxFeePrice)
+		if dynamicFee.FeeCap == nil || dynamicFee.TipCap == nil {
+			err = fmt.Errorf("estimator did not return valid fee estimates: %v", dynamicFee)
+		}
```

Comment:
This shouldn't be possible, what scenario can prompt this behavior?

Reply:
ah right - it's pulling directly from the dynamic estimator